guides/getting-started/set-up/indy-vdr.md

```diff
@@ -0,0 +1,131 @@
+# Indy VDR
+
+[Hyperledger Indy VDR](https://github.com/hyperledger/indy-vdr), Verifiable
+Data Registry, can be used to connect to one or more Indy Node ledger pools
+given sets of genesis transactions. Methods are provided to construct ledger
+requests and send them to the validators, collecting the results and ensuring
+that there is a consensus between the nodes. In the context of Aries Framework
+JavaScript, we mainly leverage it to register, and resolve, schemas, credential
+definitions and dids.
+
+:::caution
+
+Support for Indy VDR in Aries Framework JavaScript is currently experimental.
+We recommend new projects to use Indy VDR from the start, and also to migrate
+existing projects to Indy VDR, from the indy-sdk. However, projects may
+experience some issues. If you encounter any issues, please [open an
+issue](https://github.com/hyperledger/aries-framework-javascript/issues/new).
+
+Currently, there are few limitations to using Indy VDR.
+
+- React Native projects leveraging [Expo](https://expo.dev) are not supported
+  at the moment. Support for Expo will be added soon.
+- React Native Android support starts from 11 right now. Work is being done on
+  support for lower versions.
+- When running in Node.JS, only Node.JS 18 is suported for now. See [Supported
+  Node.JS versions for Indy VDR](#supported-nodejs-versions-for-indy-vdr)
+
+:::
+
+### Installing Indy VDR
+
+When using Aries Framework JavaScript with Indy VDR, there are a few extra
+dependencies that need to be installed. We need to install the
+`@hyperledger/indy-vdr`, which contains all the functionality to register
+objects on an Hyperledger Indy VDR. Secondly, we need to add native bindings
+for the specific platform `@hyperledger/indy-vdr-<platform>`. Currently there
+are bindings for Node.JS, `nodejs` and React Native, `react-native`.
+
+<!--tabs-->
+
+# Node.JS
+
+```console
+yarn add @aries-framework/indy-vdr @hyperledger/indy-vdr-nodejs
+```
+
+# React Native
+
+```console
+yarn add @aries-framework/indy-vdr @hyperledger/indy-vdr-react-native
+```
+
+<!--/tabs-->
+
+### Adding Indy VDR to the agent
+
+After installing the dependencies, we can register the Indy VDR module on the
+agent.
+
+<!--tabs-->
+
+# Node.JS
+
+```typescript showLineNumbers set-up-indy-vdr.ts section-1
+
+```
+
+# React Native
+
+```typescript showLineNumbers set-up-indy-vdr-rn.ts section-1
+
+```
+
+<!--/tabs-->
+
+As you can see, the Indy VDR module takes the native bindings and a list of
+networks. This list of networks will be used to resolve and register objects
+on. The list of networks is compatible with the older
+[`indyLedgers`](../../tutorials/agent-config/index.md#indyLedgers).
```

Comment:
I have a local PR that removes this from the docs, as the property is removed from the agent config in 0.4.0

Reply:
I added a configuration section to the document. Its mainly copied from the `indyLedgers` from the old docs. I am not too sure on the structure, but if it looks fine to you, I will also add it to aries-askar and anoncreds-rs.